assets/javascript/app.js

```diff
@@ -76,6 +94,60 @@ $(document).ready(function () {
                 //Assign title with O*Net ID for use with other ajax calls
                 occupationTitleDiv.attr("value", occupationOnetCode);
 
+                const urlOnetCode = "https://api.careeronestop.org/v1/lmi/JSoak5q9cSjVtxE/" + occupationOnetCode + location
```

Comment:
There's a couple issues I noticed. It looks like some stuff wasn't deleted when you pulled thats not in the master anymore (from here to line 130). It's showing them as additions in the search button on-click event. They were ajax calls that got moved to the on-click at the bottom (line 172). Also, it looks like you ended the for loop for the search button on-click early (line 130). The for loop is supposed to end after lines 151-166 so the divs get added for each occupation. Lastly, I think the ajax call for school by location should be moved to the bottom on-click for after they have chosen an occupation (line 172). Sorry for the long message, just didn't want you to think I was ignoring the pull. We can work it out tomorrow! Hope you had a good Saturday!